backend/internal/model/game.go

```diff
@@ -208,6 +208,16 @@ func (g *game) IncrementHeatmap(xCord float64, yCord float64) error {
 	if y > heatmapUpperBound || y < 0 {
 		return errors.New("y ball position index out of range")
 	}
+	for i := x - 1; i <= x+1; i++ {
+		for j := y - 1; j <= y+1; j++ {
+			if i > 0 && i < heatmapUpperBound {
+				if j > 0 && j < heatmapUpperBound {
+					g.gameData.Heatmap[i][j]++
+					log.Println("x: ", i, ", y: ", j)
```

Comment:
Maybe change the logging to `Trace` or `Debug` instead of `Println`?

Reply:
I overlooked it, it was unnecessary so is removed now